### ISSUE TITLE
Suggest adding a (cut-down) info object at the root

### ISSUE THREAD

**Issue by MikeRalphson** [RepreZen/SEMOASA#12]
Which might look something like:

##### Fixed Fields

Field Name | Type | Description
---|:---:|---
<a name="infoTitle"></a>title | `string` | **REQUIRED**. The title of the SEMOASA documentation.
<a name="infoDescription"></a>description | `string` | A short introduction to the documentation. [CommonMark syntax](http://spec.commonmark.org/) MAY be used for rich text representation.
<a name="infoContact"></a>contact | [Contact Object](#contactObject) | The contact information for the documented extensions.
<a name="infoVersion"></a>version | `string` | **REQUIRED**. The version (or date in ISO-8601 format) of the SEMOASA document (which is distinct from the **openapiExtensionFormat** property.

**Comment by tedepstein** [comment]
Nice idea, especially because this allows self-identification of the _publisher_, who may or may not be the same party as the _provider_.  I'm thinking that we should make that intention clear in the Description column, also allow a similar `contact` property in the Provider Object, and add a `contacts` catalog under `components` to keep it DRY.

Also, consistent with OAS3, our spec uses the term "document" to refer to the physical file containing a SEMOASA Extensions Object.

Field Name | Type | Description
---|:---:|---
<a name="infoTitle"></a>title | `string` | **REQUIRED**. The title of the SEMOASA document.
<a name="infoDescription"></a>description | `string` | A short introduction to the document. [CommonMark syntax](http://spec.commonmark.org/) MAY be used for rich text representation.
<a name="infoContact"></a>contact | [Contact Object](#contactObject) \| [Reference Object](#referenceObject)| The contact information for the publisher of the SEMOASA document.
<a name="infoVersion"></a>version | `string` | **REQUIRED**. The version (or date in ISO-8601 format) of the SEMOASA document (which is distinct from the **openapiExtensionFormat** property.

I'm on the fence as to whether `title` should be required.  Is this so we can present the source document in a UI, or in generated human-readable documentation? Or is it for some other purpose?

**Comment by MikeRalphson** [comment]
Yes, I was mainly thinking of [generated human-readable documentation](https://mermade.github.io/shins/apisguru.html). :smile:

I don't think `title` need be required.

**Comment by tedepstein** [comment]
Wow!!!  :-)

**Comment by tedepstein** [comment]
Very cool to see this, @MikeRalphson .  Did you write a generator to do this, or did [Shins](https://github.com/Mermade/shins/) auto-magically adapt to this new format?  Either way, pretty impressive. 

If there's something here that people can use today, to generate similar extension docs from a SEMOASA spec, I'd like to add it to [Implementations](https://github.com/RepreZen/SEMOASA#user-content-implementations) in the Readme.

**Comment by MikeRalphson** [comment]
I'd love to say shins could automagically derive such documentation from an arbitrary yaml file, but no, it's just a port of [slate](https://github.com/lord/slate/) to node.js - the 'clever' bit is done by three small templates added to [widdershins](https://github.com/Mermade/widdershins).

Will PR to the Readme.
